src/RTG/ChatFilter/Loader.php

```diff
@@ -33,64 +33,41 @@
 use pocketmine\event\player\PlayerChatEvent;
 
 class Loader extends PluginBase implements Listener {
-
-    public $words;
+    public $cfg;
 
     public function onEnable() {
-        $this->getServer()->getPluginManager()->registerEvents($this, $this);
-        $this->words = array();
+        $this->getServer()->getPluginManager()->registerEvents($this, $this)
         $this->cfg = new Config($this->getDataFolder() . "words.txt", Config::ENUM);
-
-        $this->words = $this->cfg->getAll();
```

Comment:
A Config object is actually a wrapper around an array similar to $this->words. Since you're using a Config class anyway, $this->words is entirely redundant.